src/pyskindose/rdsr_normalizer.py

```diff
@@ -0,0 +1,107 @@
+import pandas as pd
+from pathlib import Path
+import json
+from .settings_normalization import NormalizationSettings
+from .geom_calc import calculate_field_size
+
+
+def rdsr_normalizer(data_parsed: pd.DataFrame) -> pd.DataFrame:
+
```

Comment:
Should there be a doc string for this function

Reply:
Done. Note that this docstring will be supported bylots of illustrations in the final version.